### PR TITLE
MENU-14: Create entities

### PR DESCRIPTION
**Pull request by lmthongit98**

### CODE REVIEW

File: src/main/java/taplink/network/menu/api/common/enums/MenuTemplate.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package taplink.network.menu.api.common.enums;
+
+public enum MenuTemplate {
+    QUAN_AN,
```
Comment (hieulm2k) [inline src/main/java/taplink/network/menu/api/common/enums/MenuTemplate.java]: Let's discuss if we can use an English name for it.

File: src/main/java/taplink/network/menu/api/common/enums/StoreTemplate.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package taplink.network.menu.api.common.enums;
+
+public enum StoreTemplate {
+    QUAN_AN,
```
Comment (hieulm2k) [inline src/main/java/taplink/network/menu/api/common/enums/StoreTemplate.java]: The same as the above comment.

File: src/main/java/taplink/network/menu/api/model/City.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+public class City extends BaseEntity {
+
+    @Column(name = "CODE", nullable = false, unique = true)
+    private String code;
+
+    @Column(name = "NAME", nullable = false, unique = true)
+    private String name;
+
+    @Column(name = "SORT")
+    private Integer sort;
```
Comment (hieulm2k) [inline src/main/java/taplink/network/menu/api/model/City.java]: What does this column mean? Can we change it to isSorted and the type is boolean?

Comment (hieulm2k) [reply src/main/java/taplink/network/menu/api/model/City.java]: And the same for other same fields sort.

**Review by hieulm2k — COMMENTED** [review]
Could you check my comments about the name and type? The remaining things are ok.

**Review by nguyenngocbao — APPROVED** [review]

**Pull request merged**